package.json

```diff
@@ -2062,11 +2062,12 @@
         "lint": "eslint -c .eslintrc.js --ext .ts .",
         "lintfix": "eslint -c .eslintrc.js --fix --ext .ts .",
         "package": "ts-node ./build-scripts/package.ts",
-        "install-plugin": "vsce package -o aws-toolkit-vscode-test.vsix && code --install-extension aws-toolkit-vscode-test.vsix",
+        "install-plugin": "npm run samVisualizeCompile && vsce package -o aws-toolkit-vscode-test.vsix && code --install-extension aws-toolkit-vscode-test.vsix",
         "generateTelemetry": "node node_modules/@aws-toolkits/telemetry/lib/generateTelemetry.js --extraInput=src/shared/telemetry/vscodeTelemetry.json --output=src/shared/telemetry/telemetry.gen.ts",
         "generateConfigurationAttributes": "ts-node ./build-scripts/generateConfigurationAttributes.ts",
         "newChange": "ts-node ./build-scripts/newChange.ts",
-        "createRelease": "ts-node ./build-scripts/createRelease.ts"
+        "createRelease": "ts-node ./build-scripts/createRelease.ts",
+        "samVisualizeCompile": "webpack --config webpack.samVisualize.config.js"
```

Comment:
can this be added as a section in our existing webpack.config? why do we need a new build step?

Reply:
Likely. I'll dive deeper into the webpack.